superset-frontend/src/SqlLab/components/RunQueryActionButton/index.tsx

```diff
@@ -106,7 +105,9 @@ const RunQueryActionButton = ({
     : Button;
 
   const sqlContent = selectedText || sql || '';
-  const isDisabled = cleanSqlComments(sqlContent).length === 0;
+  const isDisabled =
```

Comment:
Maybe outside the scope of this PR, but I wonder if we should remove the `isDisable` logic as well. Regex's scare me.